app/src/main/java/com/nextcloud/talk/controllers/ChatController.kt

```diff
@@ -1615,6 +1620,66 @@ class ChatController(args: Bundle) :
                         }
                         true
                     }
+                    R.id.action_reply_privately -> {
+                        val apiVersion = ApiUtils.getConversationApiVersion(conversationUser, intArrayOf(ApiUtils.APIv4, 1))
+                        val retrofitBucket = ApiUtils.getRetrofitBucketForCreateRoom(
+                            apiVersion,
+                            conversationUser?.baseUrl,
+                            "1",
+                            message?.user?.id?.substring(6),
+                            null
```

Comment:
This conflicts with #1245 
When that is merged first we need to add a `null` before the userid